deployments/helm-chart/values.yaml

```diff
@@ -154,19 +154,8 @@ controller:
   ## Enable TLS Passthrough on port 443. Requires controller.enableCustomResources.
   enableTLSPassthrough: false
 
-  globalConfiguration:
```

Comment:
This is a convenient feature - helm allows admins to manage every aspect of KIC configuration via helm parameter, including GlobalConfiguration. Could you revert it back?

If you want to allow admins to reference the GlobalConfiguration, rather than creating it, you can add `name` parameter. So something like `globalConfiguration.name`. In this case, if `create` is `false`, Helm will configure KIC to use that Global configuration . The default name should be autogenerated as before - this is to allow installing multiple KICs in the same namespace, so that they don't have any collisions among their resources.

Reply:
Done